{{cookiecutter.project_slug}}/.neuro/live.yml

```diff
@@ -0,0 +1,158 @@
+
+defaults:
+  preset: gpu-small-p
+  life_span: 1d
+  tags:
+    - kind:flow
+    - flow-id-with-dashes:{{ cookiecutter.project_slug }}
+
+volumes:
+  data:
+    remote: storage:$[[ flow.flow_id ]]/data
+    mount: /project/data
+    local: data
+  code:
+    remote: storage:$[[ flow.flow_id ]]/{{ cookiecutter.code_directory }}
+    mount: /project/{{ cookiecutter.code_directory }}
+    local: {{ cookiecutter.code_directory }}
+  config:
+    remote: storage:$[[ flow.flow_id ]]/config
+    mount: /project/config
+    local: config
+    read_only: True
+  notebooks:
+    remote: storage:$[[ flow.flow_id ]]/notebooks
+    mount: /project/notebooks
+    local: notebooks
+  results:
+    remote: storage:$[[ flow.flow_id ]]/results
+    mount: /project/results
+    local: results
+  project:
+    remote: storage:$[[ flow.flow_id ]]
+    mount: /project
+    local: .
+
+images:
+  myimage:
+    ref: image:$[[ flow.flow_id ]]/develop
+    dockerfile: $[[ flow.workspace ]]/Dockerfile
+    context: $[[ flow.workspace ]]/
+
+jobs:
+
+  develop:
+    image: $[[ images.myimage.ref ]]
+    env:
+      EXPOSE_SSH: "yes"
+      PYTHONPATH: $[[ volumes.code.mount ]]
+    port_forward:
+      - "2211:22"
+    volumes:
+      - $[[ volumes.data.ref_ro ]]
+      - $[[ volumes.code.ref_rw ]]
+      - $[[ volumes.config.ref_rw ]]
+      - $[[ volumes.results.ref_rw ]]
+    cmd: bash
+
+  train:
+    image: $[[ images.myimage.ref ]]
+    detach: True
+    life_span: 10d
+    volumes:
+      - $[[ volumes.data.ref_ro ]]
+      - $[[ volumes.code.ref_ro ]]
+      - $[[ volumes.config.ref_ro ]]
+      - $[[ volumes.results.ref_rw ]]
+    env:
+      EXPOSE_SSH: "yes"
+      PYTHONPATH: $[[ volumes.code.mount ]]
+    bash: |
+        cd $[[ volumes.project.mount ]]
+        python -u $[[ volumes.code.mount ]]/train.py --data $[[ volumes.data.mount ]]
+
+  multitrain:
+    image: $[[ images.myimage.ref ]]
+    detach: True
+    life_span: 10d
+    volumes:
+      - $[[ volumes.data.ref_ro ]]
+      - $[[ volumes.code.ref_ro ]]
+      - $[[ volumes.config.ref_ro ]]
+      - $[[ volumes.results.ref_rw ]]
+    env:
+      EXPOSE_SSH: "yes"
+      PYTHONPATH: $[[ volumes.code.mount ]]
+    multi: true
+    bash: |
+        cd $[[ volumes.project.mount ]]
+        python $[[ volumes.code.mount ]]/train.py --data $[[ volumes.data.mount ]] $[[ multi.args ]]
+
+  jupyter:
+    image: $[[ images.myimage.ref ]]
+    http_port: 8888
+    http_auth: True
+    browse: True
+    detach: True
+    volumes:
+      - $[[ volumes.data.ref_ro ]]
+      - $[[ volumes.code.ref_rw ]]
+      - $[[ volumes.config.ref_ro ]]
+      - $[[ volumes.notebooks.ref_rw ]]
+      - $[[ volumes.results.ref_rw ]]
+    env:
+      PYTHONPATH: $[[ volumes.code.mount ]]
+    cmd: >-
+      jupyter notebook
+        --no-browser
+        --ip=0.0.0.0
+        --allow-root
+        --NotebookApp.token=
+        --notebook-dir=$[[ volumes.notebooks.mount ]]
+
+  jupyter_lab:
```

Comment:
What is the point of placing both `notebook` and `lab` modes in the same file?
Do people use both tools at the same time or select a single tool for the project?

Reply:
can we run sth like `neuro-flow run jupyter --mode=lab` ?

cc @mariyadavydova

Reply:
I think yes, but the task should contain `multi:true` and we should run
`neuro-flow run jupyter -- --mode=lab`
those `--` indicate the start of *multi* parameters.

Reply:
I think that we can simply create an action for Jupyter/JupyterLab with a parameter `mode`. I believe that people either use Jupyter Notebooks or JupyterLab inside one project.

Reply:
Added:
```
$ neuro-flow run jupyter -- notebook
$ neuro-flow run jupyter -- lab
```
AFAIK defaults are not supported.